wordpress-mobile/WordPress-Android
# Notifications add tracks events for inline actions

Fixes #20040

### Description

This PR adds missing tracks events for notification inline actions. It should have been included in #20040, but it is written generally to address the tracking for all inline actions on notifications.

-----

## To Test:


* Log in to the Jetpack app (with an account containing various notifications)
* Tap the notifications tab
* Scroll to a notification for a post like
* Expect to see a share icon
* Tap the share icon
* Expect the share activity to launch
* Expect the event to be tracked:

Logcat:

```
🔵 Tracked: notifications_inline_action_tapped, Properties: {\"inline_action\":\"SharePostButtonTapped\"}"
```

-----

## Regression Notes

1. Potential unintended areas of impact

Notifications

2. What I did to test those areas of impact (or what existing automated tests I relied on)

Manual testing

3. What automated tests I added (or what prevented me from doing so)

More refactoring is needed to add unit tests.

-----

## PR Submission Checklist:


-----

## Testing Checklist:
